Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift

```diff
@@ -425,9 +550,12 @@ extension CallManager: CXProviderDelegate {
 	}
 }
 
-extension Notification.Name {
+public extension Notification.Name {
 	public enum CallService {
 		public static let receiveCall = Notification.Name("CallService.receiveCall")
 		public static let endCall = Notification.Name("CallService.endCall")
+		public static let notification = NSNotification.Name.init("notification")
+		public static let changeTypeCall = NSNotification.Name.init("changeTypeCall")
+		public static let changeStatusBusyCall = NSNotification.Name.init("changeStatusBusyCall")
 	}
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/8649f3f2f0ce9de1db3f8c96a6cf34266ff6e2d9/Packages/ChatSecure/ChatSecure/Classes/CallKitManager/CallManager.swift#L561"></span>File should contain 500 lines or less: currently contains 561<br />
<code>file_length</code> <code>CallManager.swift:561</code></td>
    </tr>
  </tbody>
</table>